utils.js

```diff
@@ -6,7 +6,7 @@ const helpmessage =  [
   },
   {
     name: "**FAQs**",
-    value: "faq, faq-build, faq-debug, faq-keymap, glossary"
+    value: "faq, faq-build, faq-debug, faq-keymap, squeeze-avr, glossary"
   },
   {
     name: "**Building, Flashing, Drivers**",
```

Comment:
Udev should go under this section maybe?

Reply:
Added